### DISCUSSION TITLE
Unwanted forced restart

### DISCUSSION THREAD

**Discussion by zorglube** [CleverCloud/Community#44] [Add-ons]
Hello, 

I don't know if I did something wrong but, here is the situation.
I work on a prototype application with some OIDC (Spring-Boot 3 + Keycloak).
I installed a the Keycloak-ALPHA module, and confugured a Realm, with 3/5 users today, and the usage peak is 3 users (human) at a time.

Here is the problem : since the beggening I noticed that I have a to restart Keycloak every 2/3 days due to a peak of CPU consumtion and some response failing in timeout.

I understant that this message is lacking of informations. I'm open to give any help, just ping me. 

Thank for the work guys ! 
Z

**Comment by davlgd** [comment]
Can you try to upgrade the Java app to `S` flavor instead of `XS` ? we've seen some problems about it during 1st phase of alpha testing, and it could be related.

**Reply by zorglube** [reply]
For the moment it seems to be a working solution.

For anyone wondering how tp choose the scale of it,s runtime : https://www.keycloak.org/high-availability/concepts-memory-and-cpu-sizing
